pkg/schedule/prepare_checker.go

```diff
@@ -51,8 +51,8 @@ func (checker *prepareChecker) check(c *core.BasicCluster, collectWaitTime ...ti
 	}
 	notLoadedFromRegionsCnt := c.GetClusterNotFromStorageRegionsCnt()
 	totalRegionsCnt := c.GetTotalRegionCount()
-	// The number of active regions should be more than total region of all stores * collectFactor
-	if float64(totalRegionsCnt)*collectFactor > float64(notLoadedFromRegionsCnt) {
+	// The number of active regions should be more than total region of all stores * CollectFactor
+	if float64(totalRegionsCnt)*CollectFactor > float64(notLoadedFromRegionsCnt) {
 		return false
 	}
 	for _, store := range c.GetStores() {
```

Comment:
Could we use the similiar function with `isStorePrepared` to avoid to maintain the two functions?